lib/broccoli/fastboot-config.js

```diff
@@ -78,6 +82,23 @@ module.exports = class FastBootConfig extends Plugin {
     }
   }
 
+  buildConfig() {
+    // we only walk the host app's addons to grab the config since ideally
+    // addons that have dependency on other addons would never define
+    // this advance hook.
+    this.project.addons.forEach((addon) => {
+      if (addon.fastbootConfigTree) {
+        let configFromAddon = addon.fastbootConfigTree();
```

Comment:
@kratiahuja should `environment` be passed to this hook as an argument? Similar to how `environment ` is provided to ember-cli Addon `config` hook https://ember-cli.com/api/classes/Addon.html#method_config

Reply:
No I don't see that is something going to be used by addons unless you have a usecase in mind.